.github/workflows/container-image.yml

```diff
@@ -0,0 +1,57 @@
+on:
+  push:
+    branches:
+    - main
+    tags:
+    - 'v*'
+
+name: build container image
+
+jobs:
+  build:
+    name: Build container image
+    runs-on: ubuntu-latest
+    steps:
+      -
+        name: Checkout code
+        uses: actions/checkout@v2
+      -
+        name: Set up QEMU
+        uses: docker/setup-qemu-action@v1
+      -
+        name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v1
+      -
+        name: Login to GitHub Container Registry
+        uses: docker/login-action@v1
+        with:
+          registry: ghcr.io
+          username: ${{ github.repository_owner }}
+          password: ${{ secrets.GITHUB_TOKEN }}
+      -
+        name: Build and push development container image
+        if: ${{ startsWith(github.ref, 'refs/heads/') }}
```

Comment:
Is this conditional necessary? I think it's for tags only, which _would_ make it necessary, but I am trying to discern what this does 😅 . Probably my own misunderstanding.

Reply:
@mjura ?

Reply:
I will clarify with him later.